xml/monitoring.xml

```diff
@@ -111,6 +113,524 @@
+  </itemizedlist>
+  <sect2 xml:id="sec-installing-prometheus-grafana">
+   <title>Installing &prometheus; and &grafana;</title>
+   <para>
+    Install &prometheus; and &grafana; on a management server, or on a
+    separate monitoring node.
+   </para>
+   <itemizedlist>
+    <title>Prerequisites</title>
+    <listitem>
+     <para>
+      You have an installation source for &prometheus; and &grafana;:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        The packages are available from &ph;. To install &ph;, see
+        <link xlink:href="https://packagehub.suse.com/how-to-use/"/>.
+        <!-- This page is a bit out of date; could use the 'adding modules'
+         article instead when a link is available -->
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        If you have a subscription for &susemgr;, the packages are available
+        from the &susemgr; Client Tools repository.
+       </para>
+      </listitem>
+     </itemizedlist>
+    </listitem>
+   </itemizedlist>
+   <procedure xml:id="pro-installing-prometheus-grafana">
+    <title>Installing &prometheus; and &grafana;</title>
+    <para>
+     In this procedure, replace <replaceable>MNTRNODE</replaceable> with the
+     host name or IP address of the server where &prometheus; and &grafana;
+     are installed.
+    </para>
+    <step>
+     <para>
+      Install the &prometheus; and &grafana; packages:
+     </para>
+<screen>&prompt.monitor;zypper in golang-github-prometheus-prometheus grafana</screen>
+    </step>
+    <step>
+     <para>
+      Enable and start &prometheus;:
+     </para>
+<screen>&prompt.monitor;systemctl enable --now prometheus</screen>
+    </step>
+    <step>
+     <para>
+      Verify that &prometheus; works:
+     </para>
+     <stepalternatives>
+      <step>
+       <para>
+        In a browser, navigate to
+        <literal><replaceable>MNTRNODE</replaceable>:9090/config</literal>, or:
+       </para>
+      </step>
+      <step>
+       <para>
+        In a terminal, run the following command:
+       </para>
+<screen>&prompt.user;wget <replaceable>MNTRNODE</replaceable>:9090/config --output-document=-</screen>
+      </step>
+     </stepalternatives>
+     <para>
+      Either of these methods should show the default contents of the
+      <filename>/etc/prometheus/prometheus.yml</filename> file.
+     </para>
+    </step>
+    <step>
+     <para>
+      Enable and start &grafana;:
+     </para>
+<screen>&prompt.monitor;systemctl enable --now grafana-server</screen>
+    </step>
+    <step>
+     <para>
+      Log in to the &grafana; web server at
+      <literal><replaceable>MNTRNODE</replaceable>:3000</literal>.
+     </para>
+     <para>
+      Use <literal>admin</literal> for both the user name and password, then
+      change the password when prompted.
+     </para>
+    </step>
+    <step>
+     <para>
+      On the left panel, select &graf-config-icon; and click <guimenu>Data Sources</guimenu>.
```

Comment:
Suggestion: this may be tricky for people to see -- it's small -- and may cause issues with screen readers, eBook viewers and so on. How about:

"On the left panel, select the cog-wheel icon (&graf-config-icon;) and click..."

This is also a Unicode cog-wheel:
“⚙” U+2699 Gear Unicode Character
Unlike an inline image, this will scale with the font size and so on.

Reply:
Unlike an image, a Unicode gear wheel will display as a `#` character in the PDF, though, because it's unlikely to be available in our font.

Reply:
Agreed regarding screen readers etc. so I will add that in. 

However... thoughts on cog vs cog-wheel vs gear etc.?